yt_dlp/extractor/tv5mondeplus.py

```diff
@@ -15,28 +14,28 @@ class TV5MondePlusIE(InfoExtractor):
     _VALID_URL = r'https?://(?:www\.)?(?:tv5mondeplus|revoir\.tv5monde)\.com/toutes-les-videos/[^/]+/(?P<id>[^/?#]+)'
     _TESTS = [{
         # movie
-        'url': 'https://revoir.tv5monde.com/toutes-les-videos/cinema/rendez-vous-a-atlit',
-        'md5': '8cbde5ea7b296cf635073e27895e227f',
+        'url': 'https://revoir.tv5monde.com/toutes-les-videos/cinema/ceux-qui-travaillent',
```

Comment:
why are existing tests being removed?

Reply:
the old url doesn't exist anymore.

Reply:
gotcha. neither old nor new tests open for me :D